meta-mbl-bsp-common/recipes-bsp/atf/atf.inc

```diff
@@ -9,14 +9,16 @@ DEPENDS += " openssl-native coreutils-native optee-os u-boot virtual/kernel"
 
 PROVIDES += "virtual/atf"
 
+FILESEXTRAPATHS_prepend:="${THISDIR}/files:"
```

Comment:
By moving the patch to `meta-freescale-3rdparty` you have to move this line to `meta-freescale-3rdparty-mbl/recipes-bsp/atf/atf-imx6ul-pico-mbl_git.bb`.